src/locales/en.json

```
@@ -0,0 +1,254 @@
+			"description-client": "API integration PagerDuty Client ID",
+			"label-secret": "Client Secret",
+			"description-secret": "API integration PagerDuty Client Secret"
+		},
+		"delete-post": {
+			"failed": "Mattermost failed"
+		},
+		"incident-create": {
+			"title": "Create New Incident",
+			"failed": "PagerDuty service failed",
+			"impacted-label": "Impacted Service",
+			"impacted-description": "Impacted service name",
+			"title-label": "Title",
+			"title-description": "Incident title",
+			"description-label": "Description (optional)",
+			"description-description": "Incident description",
+			"assign-label": "Assign to (optional)",
+			"assign-description": "To whom this incident will be assigned",
+			"add-incident-exception": "Required data not sended [Impacted service and title]",
+			"create-incident-exception": "Required data not sended [Impacted service and title]"
+		},
+		"incident-detail": {
+			"exception": "PagerDuty get incident failed",
+			"title-decription": "Description",
+			"title-service": "Service",
+			"title-policy": "Escalation Policy",
+			"title-priority": "Priority",
+			"title-urgency": "Urgency",
+			"title-asignee": "Assignee",
+			"title-incident": "Incident: {{ url }}"
+		},
+		"incident-list": {
+			"message": "PagerDuty incident failed"
+		},
+		"list-call": {
+			"message": "PagerDuty policy failed"
+		},
+		"list-service": {
+			"message": "PagerDuty service failed"
+		},
+		"oauth": {
+			"exception-complete": "Bad Request: code param not provided",
+			"successfully-account": "You have successfully connected your PagerDuty account!",
+			"exception-account": "You still do not have connected your PagerDuty account"
+		},
+		"option": {
+			"service-failed": "PagerDuty service failed",
+			"user-failed": "PagerDuty get users failed",
+			"priorities-failed": "PagerDuty get priorities failed"
+		},
+		"reassign": {
+			"incident-failed": "PagerDuty incident failed",
+			"title": "Assign incident",
+			"header": "Choose a user to assign the incident \"{{ summary }}\" to:",
+			"user-failed": "PagerDuty get user failed",
+			"user-update": "PagerDuty incident update failed",
+			"reassign-incident": "You have reassigned incident \"{{ summary }}\" to {{ name }}"
+		},
+		"resolved": {
+			"failed": "Mattermost failed",
+			"incident-failed": "PagerDuty get incident failed",
+			"incident-exception": "\"PagerDuty incident update failed\". You already have resolved incident \"{{ summary }}\"",
+			"title-incident": "Resolve incident",
+			"header-incident": "Are you sure you want to resolve incident \"{{ summary }}\"?",
+			"incident-update-failed": "PagerDuty incident update failed",
+			"incident-update-exception": "You have succesfully resolved incident \"{{ summary }}\"!",
+			"title-trigger": "Triggered: {{ url }}",
+			"status-title": "Status",
+			"status-value": "Resolved",
+			"policity-title": "Escalation policy",
+			"incident-failed\"": "forms.resolved.incident-failed\""
```

Why have the extra quotes here?

```suggestion
			"incident-failed": "forms.resolved.incident-failed"
```